ingestion/functions/parsing/japan/japan.py

```diff
@@ -43,23 +40,48 @@ def convert_location(raw_entry):
         if term and term != "Unspecified"]
     return {"query":  ", ".join(query_terms)}
 
-
 def detect_notes(raw_notes):
     if "notes" in raw_notes:
         return raw_notes["notes"]
     else:
         return None
 
-
 def convert_date(raw_date: str):
     """
     Convert raw date field into a value interpretable by the dataserver.
-
+​
     The date filtering API expects mm/dd/YYYYZ format.
     """
-    date = datetime.strptime(raw_date, "%Y-%m-%d")
+    date = datetime.strptime(raw_date["dateAnnounced"], "%Y-%m-%d")
     return date.strftime("%m/%d/%YZ")
 
+def convert_additionalSources(additional_sourceURL: str):
```

Comment:
Total nit, but would be good to consistently use snake_case here (JSON fields which we don't control aside, of course). So `convert_additional_sources` and `additional_source_url`, e.g..